convert2rhel/backup.py

```diff
@@ -348,6 +348,125 @@ def restore(self):
         super(RestorableRpmKey, self).restore()
 
 
+class NewRestorableFile(RestorableChange):
+    def __init__(self, filepath):
+        super(NewRestorableFile, self).__init__()
+        self.filepath = filepath
+
+    def enable(self):
+        """Save current version of a file"""
+        # Prevent multiple backup
+        if self.enabled:
+            return
+
+        loggerinst.info("Backing up %s." % self.filepath)
+        if os.path.isfile(self.filepath):
+            try:
+                shutil.copy2(self.filepath, BACKUP_DIR)
+                loggerinst.debug("Copied %s to %s." % (self.filepath, BACKUP_DIR))
+
+            except (OSError, IOError) as err:
+                # IOError for py2 and OSError for py3
+                loggerinst.critical("Error(%s): %s" % (err.errno, err.strerror))
```

Comment:
This doesn't seem to have unit tests covering it. Unless I'm mistaken the Error we currently catch is for `restore()` function